packages/browser-sdk/README.md

```diff
@@ -128,6 +128,7 @@ const huddle = bucketClient.getFeature("huddle");
 // {
 //   isEnabled: true,
 //   track: () => Promise<Response>
+//   requestFeedback: () => void
```

Comment:
if we're showing the type signature, we should also include the available method arguments (or at least `...` to indicate that "arguments goes here")

Reply:
doh, great point. Updated.